Update Event Trigger Documentation (#6873)
umbraco/UmbracoDocs: 15/umbraco-cms/reference/webhooks/README.md

# Webhooks

Webhooks provide real-time, event-driven communication within Umbraco. They enable external services to react to content changes instantly by sending HTTP requests when specific events occur. This allows you to integrate with third-party services, automate workflows, and synchronize data effortlessly.

## Getting Started

To manage webhooks, navigate to **Settings > Webhooks** in the Umbraco backoffice.

![Webhooks section](images/webhook-section-v14.png)

To create a webhook, click **Create**. This opens the webhook creation screen where you can configure the necessary details.

![Creating a webhook](images/create-webhook-v14.png)

## Configuring a Webhook

### URL

The `Url` is the endpoint where the webhook will send an HTTP request when the selected event is triggered. Ensure this endpoint is publicly accessible and capable of handling incoming requests.

### Events

> Webhooks can be triggered by specific events in Umbraco. By default, Umbraco provides the following events:

Reviewer: ```suggestion
Webhooks are triggered by specific events in Umbraco. By default, the following events are available:
```

}
```

> In contrast, the `Content Deleted` event sends only the content ID:

Reviewer: ```suggestion
The `Content Deleted` event sends only the content ID:
```
